src/script/src/python/pyo3/copr_impl.rs

```diff
@@ -165,6 +164,23 @@ coprocessor = copr
 /// Cast return of py script result to `Vec<VectorRef>`,
 /// constants will be broadcast to length of `col_len`
 fn py_any_to_vec(obj: &PyAny, col_len: usize) -> PyResult<Vec<VectorRef>> {
+    // 1. check if obj is of two types:
```

Comment:
nit: looks like there is no "2."?
```suggestion
    // check if obj is of two types:
```